### PR TITLE
Create react project

### PR DESCRIPTION
**Pull request by babenko-v**

### CODE REVIEW

**Review by koliiis — APPROVED** [review]

File: View/frontend/src/pages/Registration.js
Comment (0lyaMelnyk) [inline View/frontend/src/pages/Registration.js:1]: Please keep your prs as lean as you can. Registration is a separate ticket and separate pr https://github.com/koliiis/Holy_locket/pull/5/
